### DISCUSSION TITLE
Applications - App not found error while connecting with client_id

### DISCUSSION THREAD

**Discussion by ratataplan** [fattureincloud/api#27] [Q&A] [answered]
Ho provato con due account diversi ed ottengo sempre lo stesso messaggio quando tento di collegare una nuova applicazione: "non esiste nessuna applicazione con questo client id", eppure il client ID inserito è corretto. Perché?

**Comment by ratataplan** [comment]
Some days ago I connected an application to another account in the same way and everything went fine.

**Reply by valmoz** [reply]
Hi @ratataplan 
First, I had to remove your image from the previous message, because you published your client_id in a public discussion. My suggestion is to delete the current application and create a new one from scratch.

Which kind of Authorization method are you using? in the image, you were inserting it as if it was a manual authentication.

**Comment by ratataplan** [comment]
I tried also with another account and another application and always was
not possible to connect a new application because the client id of the
application was not recognized as valid.
I had success with the account ***@***.*** but not with the accounts:
***@***.***
and also another account with different applications

Il giorno gio 28 lug 2022 alle ore 09:28 Mauro Valota <
***@***.***> ha scritto:

**Reply by valmoz** [reply]
As I said, if the app is OAuth but you're trying to insert it as manual, it doesn't work.

**Reply by ratataplan** [reply]
Yes you are right, if I use OAuth 2.0 the application will be connected automatically to the account. I missed this step.
Thank you

**Comment by valmoz** [comment, marked as answer]
> Yes you are right, if I use OAuth 2.0 the application will be connected automatically to the account. I missed this step. Thank you

(I duplicate the message to mark it as answer)

It is not possible to connect an Application like in the Manual Authentication if the App is using the OAuth flow. It will result in an "App not found" error.
